akka-stream/src/main/scala/akka/stream/impl/ConstantFun.scala

```diff
@@ -34,4 +35,6 @@ private[akka] object ConstantFun {
   val none = (_: Any) ⇒ None
 
   val two2none = (_: Any, _: Any) ⇒ None
+
+  val two2unit = (_: Any, _: Any) ⇒ ()
```

Comment:
Good catch, not used anymore (they were at some point, instead of explicit NoOp objects).
Removing